### PR TITLE
Fix in UI parts

### PR DESCRIPTION
**Pull request by PanRadaCloudBest**
some changes in 
- Recovery Send
- Password input Icon visibility
- Send Adress inputs

### CODE REVIEW

**Review by marek-siemieniuk-morawski — APPROVED** [review]

**Review by Kamill90 — APPROVED** [review]

File: src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx [hunk 1]
```diff
@@ -39,6 +48,11 @@ export class RecoveryTransactionListScreen extends PureComponent<Props, State> {
     selectedTransactions: [],
   };
 
+  componentDidMount() {
+    // sometimes we need nested this and will not touch performance
```
Comment (KedziaPawel) [inline src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx:52]: we have the same issue with the transaction list on the dashboard screen and the fix for this was: https://nyxo.app/fixing-virtualizedlists-should-never-be-nested-inside-plain-scrollviews

Comment (KedziaPawel) [inline src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx:52]: we have the same issue with the transaction list on the dashboard screen and the fix for this was: https://nyxo.app/fixing-virtualizedlists-should-never-be-nested-inside-plain-scrollviews

**Review by KedziaPawel — CHANGES_REQUESTED** [review]

File: src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx (hunk 1 above)
Comment (PanRadaCloudBest) [reply src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx]: done

File: src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx [hunk 2]
```diff
@@ -124,12 +133,68 @@ export class RecoveryTransactionListScreen extends PureComponent<Props, State> {
...
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
+        </View>
+        <View style={styles.itemWrapper}>
+          <View style={styles.transactionItemContainer}>
+            <TransactionItem onPress={toggle} item={transaction} />
+          </View>
+          <CheckBox onPress={toggle} right checked={isChecked} />
+        </View>
```
Comment (KedziaPawel) [inline src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx:172]: many duplicates

Comment (PanRadaCloudBest) [reply src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx:172]: good to catch my fault need this for make list longer :)

**Pull request closed without merging**